scripts/neighbor_advertiser

```diff
@@ -442,6 +444,8 @@ def add_mirror_acl_rule():
 
 def set_mirror_tunnel(ferret_server_ip):
     add_mirror_session(ferret_server_ip)
+    # Ensure the mirror session is created before creating the rules
+    time.sleep(DEFAULT_CONFIG_DB_WAIT_TIME)
```

Comment:
do we have retry logic in the orchagent?

Reply:
https://github.com/Azure/sonic-swss/pull/1104